### PR TITLE
Feature/wishlist

### PR DESCRIPTION
**Pull request by arijit1111**
Wishlist Pull Request

### CODE REVIEW

File: Modules/Wishlist/Database/Migrations/2023_03_09_112150_create_wishlists_table.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wishlists', function (Blueprint $table) {
+            $table->id();
+            $table->tinyInteger('board_id')->default('0');
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Database/Migrations/2023_03_09_112150_create_wishlists_table.php]: Should be unsignedBigInteger and foreign key is required to board table

File: Modules/Wishlist/Entities/Board.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 namespace Modules\Wishlist\Entities;
 
 use Illuminate\Database\Eloquent\Model;
-use Illuminate\Database\Eloquent\Factories\HasFactory;
+
 
 class Board extends Model {
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Entities/Board.php:8]: Remove additional lines below

File: Modules/Wishlist/Entities/Wishlist.php [hunk 3]
```diff
@@ -3,7 +3,7 @@
 namespace Modules\Wishlist\Entities;
 
 use Illuminate\Database\Eloquent\Model;
-use Illuminate\Database\Eloquent\Factories\HasFactory;
+
 
 class Wishlist extends Model {
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Entities/Wishlist.php:8]: Remove additional lines below

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 4]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
     {
+        $user = auth()->user();
+        if ($user->cannot('viewAny', Wishlist::class)) {
+            return response()->json([
+                'res' => false,
+                'msg' => 'User is not authorized !',
+                'data' => ""
+            ]);
+        }
         $response = $this->wishlistService->fetch($request);
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: This $request is unnecessary parameter, remove

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 5]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
         return response()->json($response);
     }
 
     /**
+     * Fetch wishlist by logged user
+     *
      * @param Request $request
      * @return JsonResponse
      */
     public function fetch(Request $request): JsonResponse
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: $request is unnecessary parameter, remove.

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 6]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
     }
 
+    /**
+     * Fetch boards which created by logged user
+     *
+     * @param Request $request
+     * @return JsonResponse
+     */
+
     public function fetchBoards(Request $request): JsonResponse
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: $request is unnecessary parameter, remove

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 7]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+        $user = auth()->user();
+
+        if ($user->cannot('viewAny', Board::class)) {
+            return response()->json([
+                'res' => false,
+                'msg' => 'User is not authorized !',
+                'data' => ""
+            ]);
+        }
         $response = $this->wishlistService->fetchBoards($request);
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: $request is unnecessary parameter, remove

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 8]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
     }
 
+    /**
+     * Add board by logged user
+     * @param WishlistRequest $request
+     * @return JsonResponse
+     *
+     */
+
     public function addBoard(WishlistRequest $request): JsonResponse
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: Should be ```BoardRequest" not "WishlistRequest"

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 9]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+        $user = auth()->user();
+        $board = Board::where('board_key', $request->key)->first();
+        if ($user->cannot('update', $board)) {
+            return response()->json([
+                'res' => false,
+                'msg' => 'User is not authorized !',
+                'data' => ""
+            ]);
+        }
+        $response = $this->wishlistService->updateBoard($request->all());
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: Should be ```$request->validated()```

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 10]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
-        return response()->json($response);
-    }
 
-    public function delete(Request $request): JsonResponse
-    {
-        $response = $this->wishlistService->delete($request);
         return response()->json($response);
     }
 
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: Remove additional space

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 11]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+                'data' => ""
+            ]);
+        }
         $response = $this->wishlistService->deleteBoard($request);
+
         return response()->json($response);
     }
 
+    /**
+     * Change board by ID
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php:197]: Add empty line below

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 12]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+                'data' => ""
+            ]);
+        }
+        $response = $this->wishlistService->updateBoard($request->all());
+
         return response()->json($response);
     }
+
+    /**
+     * Delete board by ID
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php:175]: Add empty line below

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 13]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+                'data' => ""
+            ]);
+        }
         $response = $this->wishlistService->addBoard($request->validated());
+
         return response()->json($response);
     }
+
+    /**
+     * Update board by key
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php:153]: Add empty line below

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 14]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+     * @param WishlistRequest $request
+     * @return JsonResponse
+     */
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: Remove this empty line

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 15]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+                'data' => ""
+            ]);
+        }
         $response = $this->wishlistService->fetchBoard($request);
+
         return response()->json($response);
     }
 
+    /**
+     * Add board by logged user
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php:132]: Add empty line below

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 16]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+     * @param WishlistRequest $request
+     * @return JsonResponse
+     *
+     */
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: Remove this empty line

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 17]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
         return response()->json($response);
     }
 
+    /**
+     * Get board by board key
+     *
+     * @param Request $request
+     * @return JsonResponse
+     */
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: Remove this empty line

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 18]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
         return response()->json($response);
     }
 
+    /**
+     * Fetch boards which created by logged user
+     *
+     * @param Request $request
+     * @return JsonResponse
+     */
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php]: Remove this empty lines

File: Modules/Wishlist/Http/Controllers/WishlistController.php [hunk 19]
```diff
@@ -20,63 +22,201 @@ public function __construct(WishlistService $wishlistService)
...
+                'data' => ""
+            ]);
+        }
         $response = $this->wishlistService->fetch($request);
+
+        return response()->json($response);
+    }
+
+    /**
+     * Delete wishlist by ID
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Controllers/WishlistController.php:66]: Add empty line below

File: Modules/Wishlist/Policies/BoardPolicy.php [hunk 20]
```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Modules\Wishlist\Policies;
+
+use Modules\Wishlist\Entities\Board;
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Policies/BoardPolicy.php]: Remove empty line

File: Modules/Wishlist/Policies/BoardPolicy.php [hunk 21]
```diff
@@ -0,0 +1,110 @@
...
+
+    /**
+     * Determine whether the user can view board detail.
+     *
+     * @param User $user
+     * @param Board $board
+     * @return bool
+     */
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Policies/BoardPolicy.php]: Remove empty line

File: Modules/Wishlist/Policies/BoardPolicy.php [hunk 22]
```diff
@@ -0,0 +1,110 @@
...
+        return $this->isCreator($user, $board);
+    }
+
+    /**
+     * Determine whether the user can add their board.
+     *
+     * @param User $user
+     * @return bool
+     */
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Policies/BoardPolicy.php]: Remove empty line

File: Modules/Wishlist/Policies/BoardPolicy.php [hunk 23]
```diff
@@ -0,0 +1,110 @@
...
+    public function create(User $user): bool
+    {
+        return $this->isRetailer($user);
+    }
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Policies/BoardPolicy.php]: Remove empty lines

File: Modules/Wishlist/Policies/WishlistPolicy.php [hunk 24]
```diff
@@ -0,0 +1,94 @@
...
+     *
+     * @param User $user
+     * @param Wishlist $wishList
+     * @return bool
+     */
+    public function delete(User $user, Wishlist $wishList): bool
+    {
+        return $this->isCreator($user, $wishList);
+    }
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Policies/WishlistPolicy.php]: Remove empty lines

File: Modules/Wishlist/Providers/AuthServiceProvider.php [hunk 25]
```diff
@@ -0,0 +1,37 @@
...
+
+    /**
+     * Register any authentication / authorization services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        $this->registerPolicies();
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Providers/AuthServiceProvider.php]: Remove empty line

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 26]
```diff
@@ -4,11 +4,12 @@
 
 
 use Illuminate\Support\Str;
+use Modules\Cart\Entities\Cart;
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove, unused statement

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 27]
```diff
@@ -164,108 +174,118 @@ public function add($request) {
...
     }
 
-    public function fetch($request) {
+    /**
+     * Fetch wishlist by user
+     *
+     * @param $request
+     * @return array
+     */
+    public function fetch($request): array
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove parameter and update PHPDoc

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 28]
```diff
@@ -20,76 +21,80 @@ class WishlistService
...
     protected User $user;
 
-    public function add($request) {
+    /**
+     * Add wishlist By Logged User
+     *
+     * @param $request
+     * @return array
+     */
+    public function add($request): array
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Declare parameter type like ```array $request```, more preferably name variable ```$requestData```

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 29]
```diff
@@ -276,45 +296,61 @@ public function fetch($request) {
...
     }
 
-    public function fetchBoards($request) {
+    /**
+     * Fetch Boards by logged user
+     *
+     * @param $request
+     * @return array
+     */
+    public function fetchBoards($request): array
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove parameter and update PHPDoc

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 30]
```diff
@@ -276,45 +296,61 @@ public function fetch($request) {
...
 
-    public function fetchBoard($request) {
+    /**
+     * Fetch board by Key
+     *
+     * @param $request
+     * @return array
+     */
+
+    public function fetchBoard($request): array
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Declare parameter type like array $request, more preferably name variable $requestData

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 31]
```diff
@@ -276,45 +296,61 @@ public function fetch($request) {
...
     }
 
-    public function fetchBoard($request) {
+    /**
+     * Fetch board by Key
+     *
+     * @param $request
+     * @return array
+     */
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove empty line

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 32]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
 
-    public function addBoard($request) {
+    /**
+     * Add board by user
+     *
+     * @param $request
+     * @return array
+     */
+
+    public function addBoard($request): array
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Declare parameter type like array $request, more preferably name variable $requestData

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 33]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
     }
 
-    public function addBoard($request) {
+    /**
+     * Add board by user
+     *
+     * @param $request
+     * @return array
+     */
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove empty line

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 34]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
+    public function addBoard($request): array
+    {
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove empty line

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 35]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
-            $newBoard->user_id = $request->user_id;
-            $newBoard->board_key = 'rb_' . Str::lower(Str::random(10));
-            $newBoard->name = $request->name;
-            $newBoard->visibility = $request->visibility;
-            $status = $newBoard->save();
-            if ($status) {
-                $response = ['res' => true, 'msg' => "Successfully added your board", 'data' => ''];
-            } else {
-                $response = ['res' => false, 'msg' => "Please try again!", 'data' => ''];
-            }
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove empty line

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 36]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
 
-        return $response;
     }
 
-    public function updateBoard($request) {
+    /**
+     * Update board by key
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php:420]: Add line below

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 37]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
+     * @param $request
+     * @return array
+     */
+    public function updateBoard($request): array
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Declare parameter type like array $request, more preferably name variable $requestData

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 38]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
+    {
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove additional lines

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 39]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
 
-            $board = Board::where('board_key', $request->key)->first();
-            $board->name = $request->name;
-            $board->visibility = $request->visibility;
-            $status = $board->save();
-            if ($status) {
-                $response = ['res' => true, 'msg' => "Successfully updated your board", 'data' => ''];
-            } else {
-                $response = ['res' => false, 'msg' => "Please try again!", 'data' => ''];
-            }
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove additional lines

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 40]
```diff
@@ -349,51 +385,73 @@ public function fetchBoard($request) {
...
 
-        return response()->json($response);
     }
 
-    public function deleteBoard($request) {
+    /**
+     * Delete board
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php:439]: Add line below

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 41]
```diff
@@ -403,40 +461,62 @@ public function deleteBoard($request) {
...
-            return $response;
+
+            return ['res' => true, 'msg' => 'Board successfully removed', 'data' => ""];
+
         }
-        $response = ['res' => false, 'msg' => 'Error please try again', 'data' => ""];
-        return $response;
+
+        return ['res' => false, 'msg' => 'Error please try again', 'data' => ""];
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove additional line

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 42]
```diff
@@ -403,40 +461,62 @@ public function deleteBoard($request) {
...
     }
 
-    public function delete($request) {
+    /**
+     *
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php]: Remove this line

File: Modules/Wishlist/Http/Services/WishlistService.php [hunk 43]
```diff
@@ -403,40 +461,62 @@ public function deleteBoard($request) {
...
+
+                return ['res' => false, 'msg' => "Please try again!", 'data' => ''];
+
             }
         }
-        $response = ['res' => false, 'msg' => 'Error please try again', 'data' => ""];
-        return $response;
+
+        return ['res' => false, 'msg' => 'Error please try again', 'data' => ""];
+
```
Comment (SonaKhachatryan1995) [inline Modules/Wishlist/Http/Services/WishlistService.php:487]: Remove this line

**Review by SonaKhachatryan1995 — CHANGES_REQUESTED** [review]
Remove Wishlist/Resources folder

**Review by SonaKhachatryan1995 — CHANGES_REQUESTED** [review]
Remove Wishlist/Resources folder

File: Modules/Wishlist/Database/Migrations/2023_03_09_112150_create_wishlists_table.php (hunk 1 above)
Comment (arijit1111) [reply Modules/Wishlist/Database/Migrations/2023_03_09_112150_create_wishlists_table.php]: Done

File: Modules/Wishlist/Entities/Board.php (hunk 2 above)
Comment (arijit1111) [reply Modules/Wishlist/Entities/Board.php:8]: Done

File: Modules/Wishlist/Entities/Wishlist.php (hunk 3 above)
Comment (arijit1111) [reply Modules/Wishlist/Entities/Wishlist.php:8]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 4 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 5 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 6 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 7 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 8 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 9 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 10 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 11 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php:197]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 12 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php:175]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 13 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php:153]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 14 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 15 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php:132]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 16 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 17 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 18 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 19 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php:66]: Done

File: Modules/Wishlist/Policies/BoardPolicy.php (hunk 20 above)
Comment (arijit1111) [reply Modules/Wishlist/Policies/BoardPolicy.php]: Done

File: Modules/Wishlist/Policies/BoardPolicy.php (hunk 21 above)
Comment (arijit1111) [reply Modules/Wishlist/Policies/BoardPolicy.php]: Done

File: Modules/Wishlist/Policies/BoardPolicy.php (hunk 22 above)
Comment (arijit1111) [reply Modules/Wishlist/Policies/BoardPolicy.php]: Done

File: Modules/Wishlist/Policies/BoardPolicy.php (hunk 23 above)
Comment (arijit1111) [reply Modules/Wishlist/Policies/BoardPolicy.php]: Done

File: Modules/Wishlist/Policies/WishlistPolicy.php (hunk 24 above)
Comment (arijit1111) [reply Modules/Wishlist/Policies/WishlistPolicy.php]: Done

File: Modules/Wishlist/Providers/AuthServiceProvider.php (hunk 25 above)
Comment (arijit1111) [reply Modules/Wishlist/Providers/AuthServiceProvider.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 26 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 27 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 28 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 29 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 30 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 31 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 32 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 33 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 34 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 35 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 36 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php:420]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 37 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 38 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 39 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 40 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php:439]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 41 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 42 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php]: Done

File: Modules/Wishlist/Http/Services/WishlistService.php (hunk 43 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Services/WishlistService.php:487]: Done

**Comment by arijit1111** [comment]
> Remove Wishlist/Resources folder

Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 18 above)
Comment (SonaKhachatryan1995) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: check again

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 16 above)
Comment (SonaKhachatryan1995) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: check again

Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

File: Modules/Wishlist/Http/Controllers/WishlistController.php (hunk 18 above)
Comment (arijit1111) [reply Modules/Wishlist/Http/Controllers/WishlistController.php]: Done

**Review by SonaKhachatryan1995 — APPROVED** [review]

**Pull request merged**